docs/src/full tutorials/MNIST.md

```diff
@@ -0,0 +1,96 @@
+## Image Classification Example
+An expanded version of this example, with early stopping and
+snapshots, is available [here](/examples/mnist).
+
+We define a builder that builds a chain with six alternating
+convolution and max-pool layers, and a final dense layer, which we
+apply to the MNIST image dataset.
+
+First we define a generic builder (working for any image size, color
+or gray):
+
+```julia
+using MLJ
+using Flux
+using MLDatasets
+
+# helper function
+function flatten(x::AbstractArray)
+	return reshape(x, :, size(x)[end])
```

Comment:
Those dastardly tabs that render differently than in source file editor! I wonder if Documenter can fix this?

```suggestion
    return reshape(x, :, size(x)[end])
```